pkg/apis/infrastructure/infra.go

```diff
@@ -189,43 +193,3 @@ func DisconnectInfra(projectID string, infraID string, cred types.Credentials) (
 		return DisconnectInfraData{}, err
 	}
 }
-
-// GetServerVersion fetches the GQL server version
-func GetServerVersion(endpoint string) (ServerVersionResponse, error) {
```

Comment:
Please add back this function, looks like it was deleted while resolving the conflicts